datanode/gateway/graphql/schema.graphql

```diff
@@ -4780,6 +4790,32 @@ type LiquidityProvisionsConnection {
   pageInfo: PageInfo!
 }
 
+"Information about a liquidity provider"
+type LiquidityProvider {
+  "Party ID of the liquidity provider"
+  partyId: ID!
+  "Market ID the liquidity provision is for"
+  marketId: ID!
+  "Information used for calculating an LP's fee share, such as the equity like share, average entry valuation and liquidity score, for the given liquidity provider and market"
+  feeShare: LiquidityProviderFeeShare
+}
+
+"Edge type containing the liquidity provider and cursor information"
+type LiquidityProviderEdge {
+  "Liquidity provider's information"
+  node: LiquidityProvider!
+  "Cursor for the liquidity provider"
+  cursor: String!
+}
+
+"Connection type for retrieving cursor-based paginated liquidity provider information"
+type LiquidityProviderConnection {
+  "Page of liquidity provider edges for the connection"
+  edges: [LiquidityProviderEdge]
```

Comment:
are both edges and the content of edges truly nullable? Can this be `edges: [LiquidityProviderEdge!]!` ?

Reply:
Yes, you're right, I will fix that.